conda_build/config.py

```diff
@@ -174,7 +174,13 @@ def _get_default_settings():
             Setting('extra_deps', []),
 
             # customize this so pip doesn't look in places we don't want.  Per-build path by default.
-            Setting('_pip_cache_dir', None)
+            Setting('_pip_cache_dir', None),
+
+            # set up compression algorithm used in new-style packages
+            Setting('compression_tuple', ('.tar.zst', 'zstd', 'zstd:compression-level=22')),
```

Comment:
we may want to default to using the multithreaded compression options inside zstd if we are planning on using compression level 22 which is very expensive to do.

Reply:
That's a good idea. Do you know if it affects determinism of the output, though? That and like a common trade-off, and here since the time for compression is on the build side, I think determinism will win out.

It's worth trying, to be sure.

Reply:
Looks like maybe, but it depends on the number of threads used for compression: https://github.com/facebook/zstd/issues/1077